packages/sw-background-sync-queue/test/browser/background-sync-queue.js

```diff
@@ -63,16 +63,16 @@ describe('background sync queue test', () => {
 
   it('check push proxy', () => {
```

Comment:
Consider marking this function as `async`, and then you can do

```js
await backgroundSyncQueue.pushIntoQueue({request: new Request('http://lipsum.com')});
chai.assert.equal(...);
```

See, e.g., https://github.com/GoogleChrome/sw-helpers/blob/master/packages/sw-runtime-caching/test/sw/request-wrapper.js#L214

That might make it clearer that you're not actually using the value that the first promise resolves with, and also clarify that the assertion doesn't return a value.